styles/variables.css

```diff
@@ -0,0 +1,32 @@
+:root {
```

Comment:
we may want to import that in some root file to make the variables available to be used